.travis.yml

```diff
@@ -16,7 +16,6 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-sudo: true
 dist: xenial
```

Comment:
We're already running xenial, right?

Reply:
Yep, looks like that change got through after I submitted this. I just updated this patch so that it just drops the deprecated `sudo` flag from travis.